src/components/login-form/LoginForm.tsx

```diff
@@ -54,8 +53,19 @@ function LoginForm(): JSX.Element {
           dispatch(setToken(token));
           router.push(ROUTES.home);
         })
-        .catch((error: FetchBaseQueryError) => {
-          if (error) {
+        .catch((error) => {
+          const errorMessage = error.data.message;
+          if (errorMessage === translate('loginForm.isDeletedByAdmin')) {
+            setError('password', {
+              type: 'manual',
+              message: `${translate('loginForm.isDeletedByAdmin')}`,
+            });
+          } else if (errorMessage === translate('loginForm.isInactive')) {
+            setError('password', {
+              type: 'manual',
+              message: `${translate('loginForm.isInactive')}`,
+            });
+          } else if (error) {
```

Comment:
why do we need these conditionals? You can just show an error message from server, you get different error messages from the server.

Reply:
fixed